src/components/shared/sider/sider-body.tsx

```diff
@@ -226,19 +284,38 @@ const SiderBody = () => {
 											realm.name ===
 											realmSelected?.name,
 									)[0]
-									?.userStorages.map(
-										(us) => us.name,
+									?.userStorages.map((us) =>
```

Comment:
userstorage is passed even though no userstorage has been selected as favorite us